src/components/Nav/Nav.js

```diff
@@ -1,7 +1,157 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+import styled, { css } from 'styled-components';
 
 function Nav() {
-  return <div />;
+  return (
+    <NavBar>
+      <NavBarContainer>
+        <NavLogo>
+          <Link to="/main">
+            <LogoImg src="/images/logo.svg" alt="워크 폴리오" />
+          </Link>
+        </NavLogo>
+
+        <Menu>
+          <MenuContainer>
+            <Button>
+              <MenuButtonImg
+                src="/images/location.svg"
+                alt="어디로 떠날까요?"
+              />
+              <DestinationText>어디로 떠날까요?</DestinationText>
+            </Button>
+            <Button>
+              <MenuButtonImg src="/images/calendar.svg" alt="언제 떠날까요?" />
+              <CalendarText>언제 떠날까요?</CalendarText>
+            </Button>
+          </MenuContainer>
+
+          <DetailMenu>
+            <Link to="/list-page">
+              <Text> FIND OFFICE </Text>
+            </Link>
+            <Link to="/list-page">
+              <Text> PRE-ORDER</Text>
+            </Link>
+          </DetailMenu>
+
+          <LogInContainer>
+            <Link to="/my-page">
+              <MyPage src="/images/person.svg" alt="마이 페이지" />
+            </Link>
+            <Link to="/login">
+              <LoginText>LOGIN</LoginText>
+            </Link>
+          </LogInContainer>
+        </Menu>
+      </NavBarContainer>
+    </NavBar>
+  );
 }
 
+const borderBottom = props => css`
+  border-bottom: 1px solid ${({ theme }) => theme.colorLightGray};
+`;
+
+const NavBar = styled.div`
+  padding: 0 60px;
+  ${borderBottom}
+  background-color: ${({ theme }) => theme.colorWhite};
+`;
+
+const NavBarContainer = styled.div`
+  flex: ${({ theme }) => theme.flex('row', 'center', 'space-between')};
```

Comment:
```suggestion
  ${({ theme }) => theme.flex('row', 'center', 'space-between')};
```
이렇게 써야 적용될 것 같네요!! 확인해보고 수정해주세요!

Reply:
확인해봤더니 말씀해주신 대로 수정해야 적용이 되네요! 감사합니다